geopyspark-backend/geotrellis/src/main/scala/geopyspark/geotrellis/io/LayerWriterWrapper.scala

```diff
@@ -43,17 +43,27 @@ class LayerWriterWrapper(attributeStore: AttributeStore, uri: String) {
 
   private def getTemporalIndexMethod(
     timeString: String,
+    timeResolution: String,
```

Comment:
Why `String`? Seems like this would always have to be a number, an int/long really.

Reply:
I did `String` just because it seemed easier than having to deal with both `Int` and `Long` as a possible input